torch/_inductor/choices.py

```diff
@@ -323,3 +327,69 @@ def score_fusion(
             memory_score,
             proximity_score,
         )
+
+    @staticmethod
+    def estimate_runtime(
+        counted_flops: int,
+        counted_bytes: int,
+        gpu_flops: int,
+        gpu_memory_bandwidth: int,
+    ) -> float:
+        """
+        Return estimated runtime in nanoseconds.
```

Comment:
Comment on how this is used?  How will this effect the generated code?

Last time a checked this was just for flop counter debug prints (not really a choice), but maybe it is now used in heuristics.